### PR TITLE
add more basements

### PR DESCRIPTION
**Pull request by curstwist**
#### Summary
```SUMMARY: Content "Adds basements to more houses."```

#### Purpose of change
Continue basement project.  
Includes house_08 through house_12 and associated basements.  
House_09 gets two variants for the bionic basement & its decoy.
House_08 basement is shared with house_dogs
House_10 basement is shared with house_suicide.

I also fixed bad stair placement for house_05ab basement.

#### Testing
Spawned in world.

#### Additional context
mostly basic basements.  I did expand the bionic basements to fit their house floor plan.  

House_08 & house_dogs:
house_08

bionic basements (house_09/house_09b)
bionic update
bionic decoy update

house_10 & house_suicide:
house 10

house_11
house 11

house_12
house 12

### CODE REVIEW

**Comment by I-am-Erk** [comment]
I'm so happy that the average basement is tilesetted out nicely.

File: data/json/mapgen/basement/basement_bionic.json [hunk 1]
```diff
@@ -51,64 +52,64 @@
...
+        { "item": "stereo", "x": 2, "y": 13, "chance": 50 }
       ],
       "place_monster": [ { "monster": "mon_broken_cyborg", "x": 14, "y": 3, "chance": 100 } ]
     }
   },
   {
     "type": "mapgen",
     "method": "json",
-    "om_terrain": [ "basement" ],
+    "om_terrain": [ "basement_bionic_decoy" ],
```
Comment (I-am-Erk) [inline data/json/mapgen/basement/basement_bionic.json]: If you give this the same ID, basement_bionic, and just comment on it being a decoy, you won't need separate overmap entries and things. The game will just select which map to use based on weight.

Comment (curstwist) [reply data/json/mapgen/basement/basement_bionic.json]: I did this bemuse bionic_basement is a scenario start location and I don't want people spawning in the wrong basement model.  I don't do it to other matching basements.

Comment (I-am-Erk) [reply data/json/mapgen/basement/basement_bionic.json]: Ah, yeah I didn't notice it for the others either. that makes sense.

**Pull request merged**